arm-recoveryservicesbackup/2016-12-01/swagger/backupManagement.json

```diff
@@ -18,35 +18,6 @@
     "application/json"
   ],
   "paths": {
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {
```

Comment:
This path and method signature was different from the one you replaced it with (no `/Subscriptions/...`), which is a breaking change. Are your SDKs/CLIs released? Or will you bump up the version just because of this?